src/main.ts

```diff
@@ -17,7 +17,7 @@
 /**
  * @packageDocumentation
  */
-export * as Schema from './Schema.js';
```

Comment:
let's keep it backwards-compatible and have both?

```
export * as Schema from './Schema.js';
export * from './Schema.js';
```

Reply:
makes sense, done.